contact.html

```diff
@@ -0,0 +1,161 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <meta http-equiv="X-UA-Compatible" content="ie=edge" />
+    <link
+      rel="stylesheet"
+      href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/6.3.0/css/all.min.css"
+      integrity="sha512-SzlrxWUlpfuzQ+pcUCosxcglQRNAq/DZjVsC0lE40xsADsfeQoEypE+enwcOiGjk/bSuGGKHEyjSoQ1zVisanQ=="
+      crossorigin="anonymous"
+      referrerpolicy="no-referrer"
+    />
+    <link rel="stylesheet" href="./css/main.css" />
+
+    <title>My website | Contact</title>
+  </head>
+
+  <body>
+    <div class=".container">
+      <!-- Header section -->
+      <header>
+        <nav class="nav second-nav">
+          <input
+            type="checkbox"
+            name=""
+            id="nav-checkbox"
+            class="nav-checkbox"
+          />
+          <label for="nav-checkbox" class="nav-toggle">
+            <img
+              class="menu"
+              src="./img/burger-menu.svg"
+              width="100"
+              alt="burger menu image"
+            />
+            <svg class="close" viewBox="0 0 384 512" width="100" title="times">
+              <path
+                d="M242.72 256l100.07-100.07c12.28-12.28 12.28-32.19 0-44.48l-22.24-22.24c-12.28-12.28-32.19-12.28-44.48 0L176 189.28 75.93 89.21c-12.28-12.28-32.19-12.28-44.48 0L9.21 111.45c-12.28 12.28-12.28 32.19 0 44.48L109.28 256 9.21 356.07c-12.28 12.28-12.28 32.19 0 44.48l22.24 22.24c12.28 12.28 32.2 12.28 44.48 0L176 322.72l100.07 100.07c12.28 12.28 32.2 12.28 44.48 0l22.24-22.24c12.28-12.28 12.28-32.19 0-44.48L242.72 256z"
+              />
+            </svg>
+          </label>
+          <ul class="nav-menu">
+            <li>
+              <a href="#"><div class="square"></div></a
```

Comment:
where does the # take us? If you're thinking about homepage, recommending use ./ or absolute path index.html

Reply:
# is for users to bring to the top of the current page. but in this case, the square img is already in the top of the page so user can't use it for that. typically navbar logo is to go to homepage. so in your case, absolute path <a href="index.html" .../>